deluge/plugins/Stats/deluge/plugins/stats/gtkui.py

```diff
@@ -107,21 +113,23 @@ def __init__(self, glade, colors):
         self.label.unparent()
 
         self.intervals = None
-        self.intervals_combo = self.glade.get_widget('combo_intervals')
-        cell = gtk.CellRendererText()
+        self.intervals_combo = self.main_builder.get_object('combo_intervals')
+        cell = Gtk.CellRendererText()
         self.intervals_combo.pack_start(cell, True)
         self.intervals_combo.set_cell_data_func(cell, neat_time)
         self.intervals_combo.connect("changed", self._on_selected_interval_changed)
         self.update_intervals()
 
     def graph_expose(self, widget, event):
-        context = self.graph_widget.window.cairo_create()
+        context = self.graph_widget.get_property('window').cairo_create()
         # set a clip region
-        context.rectangle(event.area.x, event.area.y, event.area.width, event.area.height)
+        alloc = Gtk.DrawingArea.get_allocation(widget)
+        w, h = alloc.width, alloc.height
+        context.rectangle(0, 0, w, h)
```

Comment:
i beleive so. if i understand correctly and remeber correctly i bleieve this just starts the graph? in the top left corner.